TINY-6037: Add `content_css_cors` setting to global `tinymce.dom.styleSheetLoader` (#8257)
tinymce/tinymce: modules/tinymce/CHANGELOG.md

- Dragging a noneditable element before or after another noneditable element now works correctly. #TINY-9253
- The restored selection after a redo or undo action was not scrolled into view. #TINY-9222
- A newline could not be inserted when the selection was restored from a bookmark after an inline `contenteditable="false"` element. #TINY-9194

> - The global `tinymce.dom.styleSheetLoader` was not affected by the `content_css_cors` setting, preventing `crossOrigin="anonymous"` from being applied to loaded stylesheets when in inline mode. #TINY-6037

Reviewer: Maybe just keep it short the reader can look up what the content_css_cors option is doing. Also it's called options now.

```suggestion
- The global `tinymce.dom.styleSheetLoader` was not affected by the `content_css_cors` option. #TINY-6037
```
